parsers/c/c_lexer.mll

```diff
@@ -529,6 +529,7 @@ and initial flags = parse
   | "^="  { CARET_EQ            }
   | "|="  { PIPE_EQ             }
   | ','   { COMMA               }
+  | "==>" { EQ_EQ_GT            }
```

Comment:
Can you please delete this, and add "implies" as a CN keyword instead? We want to avoid adding `==>` as a C token, and instead only make it visible in CN, so a keyword is the best way to proceed for now.